src/App.tsx

```diff
@@ -1,61 +1,134 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
+import { TodoWithUser } from './types/TodoWithUser';
+import { TodoList } from './components/TodoList';
+
+function getUserById(userId: number): User | null {
```

Comment:
In naming convention function that `gets` user should throw error if it has not been found.
Here it's better to use `find`

```suggestion
function findUserById(userId: number): User | null {
```